docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+---
+id: data-types
+title: Data Types Overview
+sidebar_label: Data Types and File Formats
+---
+
+ICGC ARGO provides access to harmonized data from new programs
+
+## Reads
+
+### Submitted Reads
+
+Raw reads submitted by ARGO member programs that are used as inputs to the ARGO analytic pipeline(s). Data is accepted in FASTQ or BAM format and must be compressed.
+
+#### File Types
+
+| Filename Pattern        | Description                             | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ----------------------- | --------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.fq.gz, \*.fastq.gz   | FASTQ file format compressed with gzip  | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.fq.bz2, \*.fastq.bz2 | FASTQ file format compressed with bzip2 | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.bam                  | Binary Alignment Map (BAM) file format  | sequencing_reads | Sequencing Reads | N/A                    |
+
+### Aligned Reads
+
+A read is a sequence obtained from a single sequencing experiment. An aligned read, is a sequence that has been aligned to a common reference genome. Typically these reads can number from the hundreds of thousands to tens of millions. Aligned reads are generated internally by the ARGO alignment workflow(s) and maintained in CRAM (a compressed version of BAM that stores only reads different from the reference sequence) format.
```

Comment:
remove comma here

Reply:
i wondered about linking, but will it be too much linking onthe page? to linke everytime we say analytics workflows?

Reply:
I think at least one link is helpful somewhere on the page so they know these terms are defined elsewhere

Reply:
what if i expand the intro paragraph and put it there?

Reply:
yeah that works!